lib/fluent/plugin/in_kafka.rb

```diff
@@ -174,16 +174,16 @@ def start
     opt[:min_bytes] = @min_bytes if @min_bytes
 
     if @scram_mechanism != nil && @username != nil && @password != nil
-      @kafka = Kafka.new(seed_brokers: @brokers, client_id: @client_id, logger: logger, ssl_ca_cert: read_ssl_file(@ssl_ca_cert),
```

Comment:
How about use `log` instead of removing it?
`Fluent::Plugin` has `log` method.

Reply:
Changed